GitHubRepositories.ps1

```diff
@@ -1200,6 +1222,8 @@ filter Set-GitHubRepository
     if ($PSBoundParameters.ContainsKey('NoIssues')) { $hashBody['has_issues'] = (-not $NoIssues.ToBool()) }
     if ($PSBoundParameters.ContainsKey('NoProjects')) { $hashBody['has_projects'] = (-not $NoProjects.ToBool()) }
     if ($PSBoundParameters.ContainsKey('NoWiki')) { $hashBody['has_wiki'] = (-not $NoWiki.ToBool()) }
+    if ($PSBoundParameters.ContainsKey('NoPages')) { $hashBody['has_pages'] = (-not $NoPages.ToBool()) }
```

Comment:
You may want to open a documentation bug on GitHub around this, as `has_pages` is not documented as a valid input, only as an expected output.

Reply:
Reverted. Will open a documentation bug with the GitHub API people.

Reply:
Awesome -- for tracking purposes, can you include a link to the bug report here after you've opened it?

Reply:
Checked into it, and their documentation is accurate. https://docs.github.com/en/rest/repos/repos?apiVersion=2022-11-28#update-a-repository.
That being said, I've opened an issue #378 to add Discussions support for Set-GitHubRepository (has_discussions property). I have an implementation for it once this PR is merged and closed to keep changes in my fork to a minimum.

Reply:
Right. That's their documentation is accurate and I am not 😅.